tools/jr-manifest.json

```diff
@@ -31,6 +31,11 @@
             "name": "customerimages",
             "type": "boolean",
             "description": "PROPOSED a repo involved in base and customer image creation"
+        },
+        {
+            "name": "fork",
```

Comment:
We discussed possibly using a use-case specific "nobranchprot" or "no-branch-protection" boolean label on chat.

Reply:
Based on this discussion, I think there's nothing to do for this PR: in the jira ticket, we talked about some repos that would need branch protection, but none of those are in triton.git's jr-manifest.json file, so I'm going to close this PR, and open PRs for the other repositories containing jr-manifest.json files which we do want branch protection bypasses for.